app/src/main/kotlin/com/bitwarden/authenticator/data/authenticator/manager/TotpCodeManagerImpl.kt

```diff
@@ -27,107 +22,62 @@ private const val ONE_SECOND_MILLISECOND = 1000L
  */
 class TotpCodeManagerImpl @Inject constructor(
     private val authenticatorSdkSource: AuthenticatorSdkSource,
-    dispatcherManager: DispatcherManager,
     private val clock: Clock,
 ) : TotpCodeManager {
-    private val unconfinedScope = CoroutineScope(dispatcherManager.unconfined)
 
-    private val mutableVerificationCodeStateFlowMap =
-        mutableMapOf<AuthenticatorItemEntity, StateFlow<DataState<VerificationCodeItem?>>>()
```

Comment:
This mutable map was here to support querying for a single item. Since there were no consumers of that API, it is removed!